### PR TITLE
added components

### PR DESCRIPTION
**Pull request by Beker94**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      {element}
+      <ClassComponent></ClassComponent>
```
Comment (StadnykYura) [inline src/App.js]: if there are no children - do `<ClassComponent/>`

File: src/components/Component/Component.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Component } from "react";
+
+class ClassComponent extends Component {
+  render() {
+    return (
+      <div className="ClassComponent">
```
Comment (StadnykYura) [inline src/components/Component/Component.js]: if you don't have styles for these classnames, don't write them. And which methodology are using `ClassComponent`  type of class naming?

File: src/App.js [hunk 3]
```diff
@@ -1,23 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { createElement } from "react";
+import FunctionalComponent from "./components/FunctionalComponent/FunctionalComponent";
+import Pure from "./components/PureComponent/PureComponent";
+import ClassComponent from "./components/Component/Component";
```
Comment (StadnykYura) [inline src/App.js]: bad practice for import, use 1 index file where you will export all 3 components, or create separate folder for each component and separate index file per component, or use export default

File: src/components/Component/Component.js (hunk 2 above)
Comment (Beker94) [reply src/components/Component/Component.js]: I don't use any methodology,I just named as Component

**Review by StadnykYura — APPROVED** [review]

**Pull request merged**